src/main/java/io/appform/dropwizard/actors/actor/ActorConfig.java

```diff
@@ -138,15 +129,10 @@ public int getShardCount() {
         return Objects.nonNull(shardCount) ? shardCount : 0;
     }
 
-    public int getSidelineProcessorConcurrency() {
-        return Objects.nonNull(sidelineProcessorConcurrency)
-               ? sidelineProcessorConcurrency
-               : 0;
-    }
-
     @ValidationMethod(message = "Concurrency should be multiple of shard count for sharded queue.")
     public boolean isValidSharding() {
```

Comment:
Divide this method in 2 validation method in context of sharding.

1. to validate concurrency of main queue 
2. to validate concurrency of sideline processor queue

This will help provide accurate validation error to the user while validation fails.